data/campaigns/Descent_Into_Darkness/scenarios/03_A_Haunting_in_Winter.cfg

```diff
@@ -1380,7 +1380,7 @@ All sense of time and being are entombed within darkness. How long has passed si
 
         [modify_unit]
             [filter]
-                type=Wolf,Great Wolf,Direwolf
+                type_adv_tree=Wolf
```

Comment:
Wolf unit types do not advance this way unless modified to do so. (Which they are not in DiD.)

Reply:
unless the modification takes place in a modification type extension like Reign of the Lord which offers unit_types in the extension of the mainline lines and this is the reason for this request, to allow the event to take place when a unit advances beyond the mainline without forcing addon developers to look for specific solutions limited to non-general unit_types

Reply:
excuse me, i hacen't not see what Wolf advance is null, i will fix that immediately.

Reply:
The WML still needs to work for vanilla mainline...

You can use `type_adv_tree` if you really think that is useful (do you expect a modification to add more enemy guardian units of Wolf advancements?) but it needs to include all three wolf unit types.

Reply:
Done